File: CAPITULO 02/Java/Ex08.java
import java.io.*;
import java.util.*;
import java.math.*;

public class Ex08{
	public static void main(String[]args){
		int opc,sum,n1,n2;
		double n3,raiz;
		Scanner dado = new Scanner (System.in);
		System.out.println("Selecione uma opção \n1-Para Soma\n2-Para radiciação");
		opc = dado.nextInt();
		switch(opc)
		{
				case 1:
				 System.out.println("Digite um numero ");
				 n1 = dado.nextInt();
				 System.out.println("Digite outro numero");
				 n2 = dado.nextInt();
				 sum = n1+n2;
				 System.out.println("O resultado sa soma e "+sum);
				 break;
			case 2:
				System.out.println("Digite o numero ");
				n3 = dado.nextDouble();
				raiz = Math.sqrt(n3);
				System.out.println("A raiz é"+raiz);
				break;
				default:
					System.out.println("Opção invalida");
					break;
		}
				
	}
}

File: CAPITULO 02/C++/ex08.c
#include<stdio.h>
#include <math.h>
int main(){
int opc,sum,num1,num2;
float raiz,num3;
printf("Selecione uma opcao \n1-Para somar dois numeros:  \n2-Para radiciacao: ");
scanf("%d%*c",&opc);
switch (opc){
case 1:
    printf("Digite um numero :");
    scanf("%d%*c",&num1);
    printf("Digite um numero :");
    scanf("%d%*c",&num2);
    sum = num1+num2;
    printf("A soma dos numeros e %d",sum);
    break;
case 2:
    printf("Digite um numero :");
    scanf("%f%*c",&num3);
    raiz = sqrt(num3);
    printf("A raiz e %f",raiz);
    break;
default:
    printf("Opcao invalida");
    break;
    return 0;
}
}
